### PR TITLE
datatable with list of bps

### PR DESCRIPTION
**Pull request by kar2905**

### CODE REVIEW

**Comment by noisy** [comment]
Thank you @kar2905 for your contribution! Your code looks ok, but we need to discuss internally pros and cons of adding new features to our tool.

Our priority was to focus on base functionality, to minimize a required effort of community and individuals to audit whole code.

Also, our goal was to not introduce any other dependency than official eos.js library from official eosio repository.

We also need to remember, that we already produced a video tutorial, with very detail instruction how to vote. 

Our concern is, that If someone would have 1 000 000 EOS tokens or more, and he would like to vote with that tool... and a website would differ so much from a website shown on a video... it would be at least a little bit alarming.

In other words, we really appreciate your effort, but we really need to consider all pros and cons of merging this right now.

**Comment by kar2905** [comment]
Got it. Are you okay with us pushing the modified code to the broader
network? We will obviously give proper credit to you.

**Comment by noisy** [comment]
@kar2905 our code was published under MIT licence, so you can do almost what ever you want with it.
